cli/portforward_test.go

```diff
@@ -142,21 +142,22 @@ func TestPortForward(t *testing.T) {
 		},
 	}
 
+	// Setup agent once to be shared between test-cases (avoid expensive
+	// non-parallel setup).
+	var (
+		client       = coderdtest.New(t, &coderdtest.Options{IncludeProvisionerD: true})
+		user         = coderdtest.CreateFirstUser(t, client)
+		_, workspace = runAgent(t, client, user.UserID)
+	)
```

Comment:
Note this change: We now share coderd/agent between tests. I didn't see any reason not-to.

Reply:
That's an interesting point I did not think of. We'll also be sharing the parent `*testing.T` 😮. To be fair, with regards to test logging, I rarely find them helpful due to the verbosity / parallel execution. 😂